peer dependency not fulfilled in latest release

<!-- 
Thanks for letting us know about a bug! 💖

Use the questions below to help explain what you think is wrong, and what we should do to fix it.
-->

### 👓 What did you see?

i've been getting renovate PRs for the latest release (v8.1.0). when running `npm ls` to ensure that all peer dependency requirements are met (i run this as part of my standard verification for most projects), it fails reporting:

> npm ERR! peer dep missing: @cucumber/message-streams@>=4.0.0, required by @cucumber/gherkin-streams@5.0.1

([example failure](https://github.com/form8ion/lift/runs/6100616753?check_suite_focus=true#step:5:86))

<!-- A clear and concise description of what you saw happen. -->

### ✅ What did you expect to see?

`npm ls` should exit with an exit code of zero and not report missing peer dependencies. ideally the `@cucumber/cucumber` package should declare a direct dependency on the `@cucumber/message-streams` package so that the peer requirement is not passed along to consumers of `@cucumber/cucumber`

<!-- Describe what you would like to have happened instead. -->

### 📦 Which tool/library version are you using?

`@cucumber/cucumber` v8.1.0

<!-- What version of Cucumber/GoDog/etc. are you running? What version of your programming language platform? Anything else that might be relevant? -->

### 🔬 How could we reproduce it?

* https://github.com/form8ion/lift/pull/628
* https://github.com/form8ion/lift/runs/6100616753?check_suite_focus=true#step:5:86
* if cloning the branch for that PR, simply install the proper dependencies with `npm install` and run verification with `npm test`

<!-- 
It order to fix the problem, we need to be able to reproduce it.

A Minimal Reproducible Example can be really helpful for anyone
trying to diagnose and fix the problem.

See https://stackoverflow.com/help/minimal-reproducible-example

Please outline the steps below:
-->

<!-- you can add screenshots if that would be useful -->

### 📚 Any additional context?

<!-- Add any other context, references or screenshots about the problem here. -->

----
 
*This text was originally generated from a [template](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/about-issue-and-pull-request-templates), then edited by hand. [You can modify the template here.](https://github.com/cucumber/.github/edit/main/.github/ISSUE_TEMPLATE/bug_report.md)*

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -9,6 +9,8 @@ Please see [CONTRIBUTING.md](https://github.com/cucumber/cucumber/blob/master/CO
 
 ## [Unreleased]
 
+- Capture dependency on `@cucumber/message-streams` to satisfy peer requirement from `@cucumber/gherkin-streams` ([#2006](https://github.com/cucumber/cucumber-js/pull/2006))
+
 ## [8.1.0] - 2022-04-20
 ### Added
 - Add support for named hooks (see [documentation](./docs/support_files/hooks.md#named-hooks)) ([#1994](https://github.com/cucumber/cucumber-js/pull/1994))

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -8,7 +8,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 Please see [CONTRIBUTING.md](https://github.com/cucumber/cucumber/blob/master/CONTRIBUTING.md) on how to contribute to Cucumber.
 
 ## [Unreleased]
-
+### Fixed
 - Capture dependency on `@cucumber/message-streams` to satisfy peer requirement from `@cucumber/gherkin-streams` ([#2006](https://github.com/cucumber/cucumber-js/pull/2006))
 
 ## [8.1.0] - 2022-04-20

--- a/dependency-lint.yml
+++ b/dependency-lint.yml
@@ -16,9 +16,9 @@ executedModules:
 ignoreErrors:
   missing: []
   shouldBeDependency: []
-  shouldBeDevDependency: []
-  unused:
+  shouldBeDevDependency:
     - '@cucumber/message-streams' # peer dependency of @cucumber/gherkin-streams
+  unused:
     - '@cucumber/compatibility-kit' # files dynamically loaded in cck test, not require'd
     - '@typescript-eslint/eslint-plugin' # peer dependency of standard-with-typescript
     - '@typescript-eslint/parser' # peer dependency of @typescript-eslint/eslint-plugin

--- a/dependency-lint.yml
+++ b/dependency-lint.yml
@@ -18,6 +18,7 @@ ignoreErrors:
   shouldBeDependency: []
   shouldBeDevDependency: []
   unused:
+    - '@cucumber/message-streams' # peer dependency of @cucumber/gherkin-streams
     - '@cucumber/compatibility-kit' # files dynamically loaded in cck test, not require'd
     - '@typescript-eslint/eslint-plugin' # peer dependency of standard-with-typescript
     - '@typescript-eslint/parser' # peer dependency of @typescript-eslint/eslint-plugin

--- a/package.json
+++ b/package.json
@@ -201,6 +201,7 @@
     "@cucumber/gherkin-streams": "5.0.1",
     "@cucumber/gherkin-utils": "^7.0.0",
     "@cucumber/html-formatter": "19.1.0",
+    "@cucumber/message-streams": "^4.0.1",
     "@cucumber/messages": "18.0.0",
     "@cucumber/tag-expressions": "4.1.0",
     "assertion-error-formatter": "^3.0.0",
@@ -231,7 +232,6 @@
   },
   "devDependencies": {
     "@cucumber/compatibility-kit": "9.2.0",
-    "@cucumber/message-streams": "4.0.1",
     "@cucumber/query": "11.0.0",
     "@microsoft/api-documenter": "7.17.0",
     "@microsoft/api-extractor": "7.20.0",

